tests/foreman/ui/test_reporttemplates.py

```diff
@@ -490,10 +481,12 @@ def test_negative_nonauthor_of_report_cant_download_it(session):
 
 
 @pytest.mark.tier3
+@pytest.mark.no_containers
 def test_positive_gen_entitlements_reports_multiple_formats(
```

Comment:
Looking at `org.sca_disable()` bellow, is this test case really applicable to 6.16? Shouldn't it be raised against `6.15.z` branch instead?

Reply:
This change fixes the test in 6.15 and will be removed in 6.16. however, a lot of the other work in this PR still applies to 6.16. so I will just create a separate PR to remove this later in 6.16 as part of the RFE for SCA ONLY